### PR TITLE
Docs fixes

### PR DESCRIPTION
**Pull request by munozemilio**
Fixes: #238, #237, #229, #226, #227, #234

### CODE REVIEW

File: packages/chatdown/README.md [hunk 1]
```diff
@@ -20,32 +20,26 @@ USAGE
   $ bf chatdown
 
 OPTIONS
-  -i, --in=in                  The path of the chat file or directory to be parsed. A glob expression may be passed containing 
-                               chat files to be processed all at once, ex. ./**/*.chat. If flag is omitted, stdin will be used. 
-                               If an output directory is not present (-o), it will default the output to the current working directory.
+  -f, --force    If --out flag is provided with the path to an existing file, overwrites that file
```
Comment (scheyal) [inline packages/chatdown/README.md]: Just verifying, if not provided, return an error message if exists?

Comment (munozemilio) [reply packages/chatdown/README.md]: if --out is not specified the stdout is used

Comment (munozemilio) [reply packages/chatdown/README.md]: if --force is not set then the file is numerated as filename(1).ext

File: packages/cli/README.md [hunk 2]
```diff
@@ -291,148 +286,86 @@ USAGE
...
 
 OPTIONS
-  --in=in                      Source .lu file(s) or LUIS application JSON model
-  --out=out                    Output folder name. If not specified stdout will be used as output
-  --recurse                    Indicates if sub-folders need to be considered to file .lu file(s)
+  -f, --force                  If --out flag is provided with the path to an existing file, overwrites that file
+  -h, --help                   luis:translate help
+  -i, --in=in                  Source .lu file(s) or LUIS application JSON model
+  -o, --out=out                Output folder name. If not specified stdout will be used as output
+  -r, --recurse                Indicates if sub-folders need to be considered to file .lu file(s)
```
Comment (scheyal) [inline packages/cli/README.md]: Review "to file..."

**Pull request merged**